### PR TITLE
feat(words): add words which are listed in awf/autoware.universe #2216 ( 1st shot )

### PR DESCRIPTION
**Pull request by HansRobo**
Reference: https://github.com/autowarefoundation/autoware.universe/pull/2216

## words 
| word | category | detail / evidence | 
| --- | --- | --- | 
| IROS | conference name | https://www.ieee-ras.org/conferences-workshops/fully-sponsored/icra |
| ICRA | conference name | https://www.ieee-ras.org/conferences-workshops/financially-co-sponsored/iros |
| Waymo | company name | https://waymo.com/ | 
| MORAI | company name | https://www.morai.ai/ |
| rrtstar | algorithm name | RRT*  |
| aabb | abbreviation | AABB : axis-aligned bounding boxes, common abbreviation |
| RRT | algorithm name | Rapidly exploring Random Tree |
| demangling | term in computer science | https://gcc.gnu.org/onlinedocs/libstdc++/manual/ext_demangling.html | 
| bluez | command name | https://packages.ubuntu.com/focal/bluez |
| bluetoothctl | command name | https://manpages.ubuntu.com/manpages/xenial/man1/bluetoothctl.1.html |
| hcitool | command name | https://manpages.ubuntu.com/manpages/focal/en/man1/hcitool.1.html |
| rstar | library module name | boost::geometry::index::rstar |
| binarized | word | binalize + ed commonly used in image processing |
| MODELZOO | term? | a word means collection of machine learning models https://modelzoo.co  |
| libopenblas | library name | https://packages.ubuntu.com/focal/libopenblas-base |
| mipmap | term | https://en.wikipedia.org/wiki/Mipmap |

## patterns
- ignore bibtex in markdown
- ignore references in source code

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@HansRobo Thanks, but please update `/.cspell.json` instead!
image

**Comment by HansRobo** [comment]
OK! I will fix soon

**Comment by kenji-miyake** [comment]
Also, could you provide links for each word like this?
Waymo: https://waymo.com/
MORAI: https://www.morai.ai/

And please use the exact expressions for proper nouns. In this case, Waymo, MORAI, etc.

**Comment by kenji-miyake** [comment]
> Karaman	person name

For person names, please consider using `cspell: ignore Name`.
Or you can add ignore patterns if it's possible.

**Comment by kenji-miyake** [comment]
Or it might be good to create a dictionary `person-names` in https://github.com/tier4/cspell-dicts and put names there.

**Comment by HansRobo** [comment]
I haven't been able to work on this pull request much lately, so I'd like to merge once.
I will create the second shot pull request for #2216 later

File: .cspell.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
         "\\]\\(#.+?\\)",
         "^\\| GPU  Name.+$",
         "^gdown .+$",
+        "```bibtex[\\s\\S]*```",
```
Comment (kenji-miyake) [inline .cspell.json:68]:
```suggestion
```

File: .cspell.json [hunk 2]
```diff
@@ -172,6 +173,7 @@
       "filename": "**/{*.cpp,*.hpp}",
       "ignoreRegExpList": [
         "@author .*$",
+        "[Ref,ref]\\s*:.*$",
```
Comment (kenji-miyake) [inline .cspell.json:176]:
```suggestion
```

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**